modules/oe_translation_poetry/config/install/core.entity_form_display.oe_translation_request.poetry_translation_request.default.yml

```diff
@@ -0,0 +1,41 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.oe_translation_request.poetry_translation_request.oe_contact_information
+    - field.field.oe_translation_request.poetry_translation_request.oe_organisation_information
+    - field.field.oe_translation_request.poetry_translation_request.oe_poetry_request_id
+    - field.field.oe_translation_request.poetry_translation_request.oe_translation_deadline
+    - field.field.oe_translation_request.poetry_translation_request.oe_update_of
+    - oe_translation.oe_translation_request_type.poetry_translation_request
+  module:
+    - datetime
+    - oe_translation
+    - oe_translation_poetry
+id: oe_translation_request.poetry_translation_request.default
+targetEntityType: oe_translation_request
+bundle: poetry_translation_request
+mode: default
+content:
+  oe_contact_information:
+    weight: 17
+    settings: {  }
+    third_party_settings: {  }
+    type: oe_translation_poetry_personal_contact_widget
+    region: content
+  oe_organisation_information:
+    weight: 23
+    settings: {  }
+    third_party_settings: {  }
+    type: oe_translation_poetry_organisation_contact_widget
+    region: content
+  oe_translation_deadline:
+    weight: 5
+    settings: {  }
+    third_party_settings: {  }
+    type: datetime_default
+    region: content
+hidden:
+  oe_poetry_request_id: true
+  fiel  d_update_of: true
```

Comment:
This is a field that you probably renamed and it should be replaced by "oe_update_of".

Reply:
![image](https://user-images.githubusercontent.com/22908988/86579645-e01c7500-bf7d-11ea-8052-8ea6473c7e26.png)

Reply:
Fixed.